src/scales/scale.time.js

```diff
@@ -45,8 +58,17 @@ module.exports = function(Chart) {
 				throw new Error('Chart.js - Moment.js could not be found! You must include it before Chart.js to use the time scale. Download at https://momentjs.com');
 			}
 
+			this.mergeTicksOptions();
+
 			Chart.Scale.prototype.initialize.call(this);
 		},
+		mergeTicksOptions: function() {
```

Comment:
The docs for this function imply that it should be applicable to all scales. Should `mergeTicksOptions` be moved to the base scale class?